### PR TITLE
 Add PRIA space

### PR DESCRIPTION
**Pull request by dahifi**
**Add space for PRIA **  
  
Etherscan: https://etherscan.io/address/0xb9871cb10738eada636432e86fc0cb920dc3de24
CoinGecko: https://www.coingecko.com/en/coins/pria
Website: https://pria.network/
Twitter: https://twitter.com/defi_labs_

### CODE REVIEW

File: skins/pria.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.example {
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (gokaiorg) [inline skins/pria.scss:10]: If you use the default theme you will have to delete the scss file.

File: spaces/pria/index.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "token": "0xb9871cb10738eada636432e86fc0cb920dc3de24",
+  "name": "PRIA ",
+  "network": "1",
+  "symbol": "PRIA",
+  "skin": "pria",
```
Comment (gokaiorg) [inline spaces/pria/index.json:6]: If you use the default theme you will have to remove the skin field.

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Review by gokaiorg — CHANGES_REQUESTED** [review]
We've made a major global update, please pull before commit again your changes.
You can use the documentation to help you.
https://docs.snapshot.page/guides/create-a-space#1-fork-snapshot-spaces-repository-here

**Comment by gokaiorg** [comment]
Your PR seem to be a duplicate with #406

**Comment by dahifi** [comment]
@gokaiorg, I merged your global commits and removed the default scss templates from both locations. We should be G2G, and will get with our team on cancelling https://github.com/snapshot-labs/snapshot-spaces/pull/406

Thanks!

File: spaces/pria/index.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "token": "0xb9871cb10738eada636432e86fc0cb920dc3de24",
+  "name": "PRIA ",
```
Comment (bonustrack) [inline spaces/pria/index.json:3]:
```suggestion
  "name": "PRIA",
```

**Pull request merged**